### ISSUE TITLE
Add-on Submissions 22W51

### ISSUE THREAD

**Issue by DaveBowman2001** [levinli303/celestia.mobi#46]
Submit your add-ons in this issue, thanks for your contribution!
Submission template:
```
**Title:**
**ID:** (optional, use only to determine which addon in mobi to update)
**ID Requirement:** (optional, ID decides the order of loading, might need special requirement if an add-on has dependencies)
**Description (Plain Text)**: (plain text description of the add-on)
**Description (Rich)**: (optional, rich content description of the add-on which includes additional images, videos, tables etc. which also replaces the old Additional metadata format. If it is plain text, put it in Description (Legacy) and leave this empty)
**Proposed category:** (pick one of the existing categories (https://celestia.mobi/resources/categories), or suggest a new one)
**Demo Object Name:** (optional, the name of the main object used for Go to command)
**Author(s):** (optional)
**Release date:** (optional, The date this original add-on is released. If left empty, will be the date the add-on is uploaded here)
**Last Updated:** (optional, only for updating addons to keep track of their versions especially if it is regularly updated)
**Link to download:** (Please use zip to package the add-on, it should NOT contain a file called description.json)
**Image:** (optional, no UI control element, HUD display, label. Avoid using "tall" pictures (height > width))
**Disable comments**: (optional, by default, comments are not disabled)
```
## For Updates to Existing Add-ons
For updates to existing add-ons, please attach the id to the add-on and only include updated fields from the template.
For removal of add-ons, please attach the id too.

The id is part of the link to the add-on:
For example, this addon below has an id of A4577B17-4C60-0C1E-CAB5-92A604FF0EC3
https://celestia.mobi/resources/item?item=A4577B17-4C60-0C1E-CAB5-92A604FF0EC3

## **Check Before Uploading an Add-on**

Your add-on will be running on Android/iOS/macOS/Windows, to enable maximum compatibility, please ensure that:

1. File names referenced in **models**, **ssc**, **dsc** files, etc. are exactly the same as the actual file names (some file systems are case sensitive. For example if in your .ssc there is `Mesh "Progress.cmod"`, then the file name should be exactly `Progress.cmod` not `progress.cmod`).
2. Textures should have width/height of power of 2 (for example: 1024 * 512, a texture of 1000 * 500 will not work on many mobile devices).
3. If your add-on requires specific Celestia versions, please mention in the description. Please also ensure that your add-on should not crash running on other versions.
4. If you are updating an existing add-on, note that existing users will not see an update button, so it might be a good idea to include version history in the description.
5. Please try to use smaller files for image/thumbnail/other metadata. For example, use JPG if alpha value is not needed.
6. All images need to be in JPEG (you either choose to screenshot in JPEG in Celestia or convert using tools)
7. The add-on archive should not include system files or hidden files (macOS is notorious for adding hidden system files to archives, avoid using macOS's default archive utility) this issue is found in many add-ons @Anthony-B-Russo10 uploaded
8. The add-on archive must have a flat folder structure (It should try to have fewer level of folders, do not do things like A/A/A) this issue is found in many add-ons @SpaceExplorer2008 uploaded

**Example**:

**Title**: Kepler-1649
**Description (Legacy)**: The Kepler-1649 planetary system, which is a red dwarf star orbited by a Venus-analogue and an Earth-analogue. Alternate surfaces for planet c are included.
**Proposed category**: Exoplanets
**Demo Object Name**: Kepler-1649
**Author(s)**: SevenSpheres
**Release date**: 2020-Jul-16
**Link to download**: [Kepler-1649.zip](https://drive.google.com/file/d/1KkyuH2LkYCw6mXnG4by-TzLxccsN9SxF/view?usp=sharing)
**Image**:
![Untitled](https://user-images.githubusercontent.com/56284207/148109832-5e2e8df2-dac6-4fc2-b063-978c3088412f.png)

**Comment by DaveBowman2001** [comment]
Update to the "**S-IVB Apollo stages**" addon. Authors correction and modified model/texture by pedro_jg in preparation for the upcoming _Project Apollo (Part 4)_

**ID:** [106292B4-8C00-E26E-2AA7-24DE5DB680E3](https://celestia.mobi/resources/item?item=106292B4-8C00-E26E-2AA7-24DE5DB680E3)
**Description (Plain Text):** This addon adds the S-IVB stages of the Saturn 1B and Saturn V rockets that propelled the Apollo missions to LEO, Skylab, and the Moon. About 7 out of 21 stages are included on this addon

NOTE: The names of these spacecraft are permanently hidden to simulate their status as inactive, defunct objects floating in space, but they can still be searched on Celestia itself upon entering the word "S-IVB"

**Description:** This addon adds the S-IVB stages of the Saturn 1B and Saturn V rockets that propelled the Apollo missions to LEO, Skylab, and the Moon. About 7 out of 21 stages with known orbital data are included on this addon:
- 1966-059A = AS-203
- 1968-089B = AS-205 (Apollo 7)
- 1968-118B = AS-503 (Apollo 8)
- 1969-018B = AS-504 (Apollo 9)
- 1969-043B = AS-505 (Apollo 10)
- 1969-059B = AS-506 (Apollo 11)
- 1969-099B = AS-507 (Apollo 12), later rediscovered as J002E3

NOTES:
- The addon's Go-to Object button will take you to the S-IVB stage of Apollo 12, which was rediscovered in 2002 as J002E3 and is currently in heliocentric orbit
- Only the S-IVB stages that are currently in heliocentric orbit (from Apollo 8 to 12) can be visited on the Present Day, as most of them have either burned up on the atmosphere or deliberately crashed onto the Moon
- The names of these upper stages are permanently hidden to simulate their status as inactive, defunct objects floating in space, but they can still be searched on Celestia itself upon entering the word "S-IVB" or "AS-"

![Untitled](https://user-images.githubusercontent.com/95486841/205529477-a01bba11-05ee-4e55-bcf5-2903283f3a57.jpg)

**Author(s):** DaveBowman2001, Medusa, pedro_jg
**Release date:** 2005-Dec-12
**Last Updated:** 2022-Dec-4
**Link to download:** [S-IVB Apollo stages.zip](https://github.com/levinli303/celestia.mobi/files/10149985/S-IVB.Apollo.stages.zip)
**Image:**
![Untitled](https://user-images.githubusercontent.com/95486841/205529477-a01bba11-05ee-4e55-bcf5-2903283f3a57.jpg)

**Comment by DaveBowman2001** [comment]
Update/Proposal to rename the "**2020 SO**" addon into "**Centaur-D**". Added 20 other stages with the same design as 2020 SO

**Title:** Centaur-D
**ID:** [009B34FB-F229-F520-6EE4-15B93F6EF42F](https://celestia.mobi/resources/item?item=009B34FB-F229-F520-6EE4-15B93F6EF42F)
**Description (Plain Text)**: This addon adds the spent upper stages called the Centaur-D booster, which were in active service from 1962 to 1989. As of November 2022, only 21 such stages are added with known orbital data, More objects will be added in the near future

NOTE: The names of these spacecraft are permanently hidden to simulate their status as inactive, defunct objects floating in space, but they can still be searched on Celestia itself upon entering the word "Centaur"

**Description (Rich)**: This addon adds the spent upper stages called the Centaur-D booster, which were in active service from 1962 to 1989. As of November 2022, a total of 21 objects were available below, more will be added in the near future:
- 1963-047A = AC-2
- 1966-084B = AC-7 (Surveyor 2), later rediscovered as 2020 SO
- 1968-110B = AC-16 (OAO 2)
- 1969-069B = AC-18 (ATS 5)
- 1971-006B = AC-25 (Intelsat 402)
- 1971-116B = AC-26 (Intelsat 403)
- 1972-003B = AC-28 (Intelsat 404)
- 1972-041B = AC-29 (Intelsat 405)
- 1972-065B = AC-22 (OAO 3)
- 1973-058B = AC-31 (Intelsat 407)
- 1974-093B = AC-32 (Intelsat 408)
- 1975-042B = AC-35 (Intelsat 401)
- 1975-091B = AC-36 (Intelsat 4A-1)
- 1976-010B = AC-37 (Intelsat 4A-2)
- 1976-042B = AC-38 (Comstar 1A)
- 1976-073B = AC-40 (Comstar 1B)
- 1978-035B = AC-48 (Intelsat 4A-6)
- 1978-068B = AC-41 (Comstar 1C)
- 1981-018B = AC-42 (Comstar 1D)
- 1985-055B = AC-64 (Intelsat 5A-11)
- 1985-087B = AC-65 (Intelsat 5A-12)

NOTES:
- The addon's Go-to Object button will take you to the Centaur booster of the [Surveyor 2](https://celestia.mobi/resources/item?item=87F11016-16D9-0411-9149-6EA69F8CDF88) spacecraft, also known as 2020 SO
- The names of these upper stages are permanently hidden to simulate their status as inactive, defunct objects floating in space, but they can still be searched on Celestia itself upon entering the word "Centaur" or "AC-"

![Untitled2](https://user-images.githubusercontent.com/95486841/203294457-cabb4cd2-c2f8-4fd0-b2bc-410b51085e5d.jpg)

**Author(s):** Thomas Guilpain, Art Blos, DaveBowman2001
**Last Updated:** 2022-Nov-22
**Link to download:** [Centaur-D.zip](https://celestia.space/forum/download/file.php?id=14785&filename=Centaur-D.zip)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/95486841/203294457-cabb4cd2-c2f8-4fd0-b2bc-410b51085e5d.jpg)

**Comment by DaveBowman2001** [comment]
**Title:** Centaur-III(5)
**ID Requirement:** B
**Description (Plain Text)**: This addon adds the spent upper stages of missions launched from an Atlas V rocket called the Centaur-III(5) booster. As of November 2022, only 40 such stages are added with known orbital data, More objects will be added in the near future

NOTE: The names of these spacecraft are permanently hidden to simulate their status as inactive, defunct objects floating in space, but they can still be searched on Celestia itself upon entering the word "Centaur"

**Description (Rich)**: This addon adds the spent upper stages of missions launched from an Atlas V rocket called the Centaur-III(5) booster. As of November 2022, a total of 40 objects were available below, more will be added in the near future:
- 2002-038B = AV-001 (Hot Bird 6)
- 2003-033B = AV-003 (EchoStar 12)
- 2004-048B = AV-005 (AMC-16)
- 2005-029B = AV-007 (MRO)
- 2006-012B = AV-008 (Astra-1KR)
- 2007-060B = AV-015 (NROL-24)
- 2009-017B = AV-016 (WGS-2)
- 2009-031C = AV-020 (LRO and LCROSS), crashed onto the lunar surface
- 2009-047B = AV-018 (PAN)
- 2009-064B = AV-024 (Intelsat 14)
- 2010-005B = AV-021 (SDO)
- 2010-039B = AV-019 (AEHF-1)
- 2011-040B = AV-029 (Juno)
- 2011-070B = AV-028 (MSL)
- 2012-009B = AV-030 (MUOS-1)
- 2013-004B = AV-036 (TDRS-11)
- 2013-023B = AV-039 (GPS-2F4)
- 2013-036B = AV-040 (MUOS-2)
- 2014-004B = AV-043 (TDRS-12)
- 2014-020B = AV-045 (NROL-67)
- 2014-045B = AV-048 (GPS-2F7)
- 2014-055B = AV-049 (CLIO)
- 2014-068B = AV-051 (GPS-2F8)
- 2015-002B = AV-052 (MUOS-3)
- 2015-033B = AV-055 (GPS-2F10)
- 2015-044B = AV-056 (MUOS-4)
- 2015-056B = AV-059 (Mexsat-2)
- 2015-062B = AV-060 (GPS-2F11)
- 2016-041B = AV-063 (MUOS-5)
- 2016-071B = AV-069 (GOES-16)
- 2017-047B = AV-074 (TDRS-13)
- 2018-022B = AV-077 (GOES-17)
- 2018-079B = AV-073 (AEHF-4)
- 2019-051C = AV-083 (AEHF-5)
- 2020-022C = AV-086 (AEHF-6)
- 2021-093B = AV-096 (Lucy), briefly mistaken for an asteroid called 2022 UQ1
- 2021-118C = AV-093 (STP-3)
- 2022-021B = AV-095 (GOES-18)
- 2022-092B = AV-097 (SBIRS-GEO6)
- 2022-123C = AV-099 (SES-20 and 21)

NOTES: 
- The addon's Go-to Object button will take you to the Centaur booster of the [Lucy](https://celestia.mobi/resources/item?item=570B15DB-AAE1-5BF0-BDD1-4E0639CDEB32) spacecraft
- The names of these upper stages are permanently hidden to simulate their status as inactive, defunct objects floating in space, but they can still be searched on Celestia itself upon entering the word "Centaur" or "AV-"
- The Centaur from LRO/LCROSS requires the [LCROSS](https://celestia.mobi/resources/item?item=A79B502E-E959-86EE-4155-1D881B841006) addon to be installed first to be viewed, which is only visible from June 18 to October 9, 2009

![Untitled2](https://user-images.githubusercontent.com/95486841/203290866-cbea9084-68d9-4dea-a4e5-de9445307ff9.jpg)

**Proposed category:** Spacecraft: Space Debris
**Demo Object Name:** 2021-093B
**Author(s):** DaveBowman2001, EarthMoon, BrianJ, toutatis
**Release date:** 2022-Sept-18
**Last Updated:** 2022-Nov-22
**Link to download:** [Centaur-3(5).zip](https://celestia.space/forum/download/file.php?id=14789&filename=Centaur-3%285%29.zip)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/95486841/203290866-cbea9084-68d9-4dea-a4e5-de9445307ff9.jpg)

**Comment by DaveBowman2001** [comment]
From JVV's "*[just a random Earthlike planet with a NormalMap](https://discord.com/channels/558624707147071489/558643440422879232/1042182734124154971)*" on Discord

**Title:** Kepler-61
**Description (Plain Text)**: Kepler-61, also known as KOI-1361, is a K-type main sequence star about 1,103 light-years away in the constellation Cygnus. In 2013, a planet 2.15 times larger than Earth was discovered orbiting the star's habitable zone. However, subsequent analyses indicate that this planet has a very eccentric orbit that takes it outside the stellar HZ. These characteristics, in addition to now theoretically believed to be closer in composition to a warm neptunian rather a superterran, has led to its removal from the Habitable Exoplanets Catalog in 2021

**Description (Rich)**: Kepler-61, also known as KOI-1361, is a K-type main sequence star about 1,103 light-years away in the constellation Cygnus. In 2013, a planet 2.15 times larger than Earth was discovered orbiting the star's habitable zone. However, subsequent analyses indicate that this planet has a very eccentric orbit that takes it outside the stellar HZ. These characteristics, in addition to now theoretically believed to be closer in composition to a warm neptunian rather a superterran, has led to its removal from the Habitable Exoplanets Catalog in 2021

NOTES:
- The planet's texture depicts it as Earth-like, following its discovery report in 2013
- Two alternate versions of the planet's map are included, the default one by John van Vliet, and the other by 星颜Crazy

![Untitled2](https://user-images.githubusercontent.com/95486841/204430912-db71991e-d6fa-4936-a4d5-b0104deca172.jpg)
**Proposed category:** Exoplanets: K
**Demo Object Name:** Kepler-61
**Author(s):** John van Vliet, 星颜Crazy
**Release date:** 2022-Sept-18
**Last Updated:** 2022-Nov-16
**Link to download:** [Kepler-61.zip](https://1drv.ms/u/s!Am77jIlPZx-khTuJ8PW4Vm_YJsKy?e=bKFv41)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/95486841/204430912-db71991e-d6fa-4936-a4d5-b0104deca172.jpg)

**Comment by DaveBowman2001** [comment]
**Title:** SDSS J122859.93+104032.9
**Description (Plain Text)**: Also known as SDSS J1228+1040, it is a white dwarf star about 413.33 light-years away in the constellation Virgo. It is home to one of only two known asteroids discovered outside our Solar System, estimated to be about 126 km in diameter
**Proposed category:** Exoplanets: Q, R and S
**Demo Object Name:** SDSS 1228+1040
**Author(s):** Gurren Lagann
**Release date:** 2022-Nov-7
**Link to download:** [SDSS 1228+1040.zip](https://1drv.ms/u/s!Am77jIlPZx-khT0ZgraTdZShUAnA?e=xgHX30)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/95486841/204440479-0c1dfd5d-c122-40e1-8bc9-677ebcd947df.jpg)

**Comment by DaveBowman2001** [comment]
**Title:** MOA-2010-BLG-477L
**Description (Plain Text)**: MOA-2010-BLG-477L is a white dwarf star about 6,491 light-years away in the constellation Sagittarius. One superjovian planet orbiting at a distance of 2.79 AU from its star was discovered through microlensing
**Proposed category:** Exoplanets: M and N
**Demo Object Name:** MOA-2010-BLG-477L
**Author(s):** Gurren Lagann
**Release date:** 2022-Nov-7
**Link to download:** [MOA-2010-BLG-477L.zip](https://1drv.ms/u/s!Am77jIlPZx-khT4CCSZ6puQCUKgD?e=v6YruB)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/95486841/204443553-0b13559d-e8e9-46c8-8b70-493fe575b5a3.jpg)

**Comment by DaveBowman2001** [comment]
**Title:** WD J0914+1914
**Description (Plain Text)**: WD J0914+1914 is a white dwarf star about 1,504 light-years away in the constellation Cancer.  It is surrounded by a compact spiral protoplanetary disk and a planet about 3/4 the size of Neptune
**Description (Rich)**: WD J0914+1914 is a white dwarf star about 1,504 light-years away in the constellation Cancer.  It is surrounded by a spiral protoplanetary disk and a planet about 3/4 the size of Neptune

![Untitled2](https://user-images.githubusercontent.com/95486841/204435088-1cab40af-d409-4622-a242-665386704a39.jpg)
_The debris disk around WD J0914+1914_

![Untitled2](https://user-images.githubusercontent.com/95486841/204435135-9501806b-e2b2-451f-a189-8576513f77fa.jpg)
_The planet WD J0914+1914 b is only 10 million km away from its dead host star_


**Proposed category:** Exoplanets: 
**Demo Object Name:** WD J0914+1914
**Author(s):** Gurren Lagann, Vision, Cham
**Release date:** 2022-Nov-7
**Link to download:** [WD J0914+1914.zip](https://1drv.ms/u/s!Am77jIlPZx-khT_MQYwjLKNTmtXK?e=E1b7BD)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/95486841/204435088-1cab40af-d409-4622-a242-665386704a39.jpg)

**Comment by DaveBowman2001** [comment]
**Title:** Kepler-44
**Description (Plain Text)**: Kepler-44, also known as KOI-204, is a G-type subgiant star about 4,009 light-years away in the constellation Cygnus. A superjovian planet completes an orbit once every 3.25 days at a distance of 6.67 million km from its star
**Proposed category:** Exoplanets: K
**Demo Object Name:** Kepler-44
**Author(s):** 星颜Crazy
**Release date:** 2022-Sept-24
**Link to download:** [Kepler-44.zip](https://celestia.space/forum/download/file.php?id=14370&filename=Kepler-44.zip)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/95486841/204442422-f72cff25-5b1f-4e7c-8139-fe32092c527b.jpg)

**Comment by DaveBowman2001** [comment]
**Title:** KOI-2700
**Description (Plain Text)**: KOI-2700 is a K-type main sequence star about 1,669 light-years away in the constellation Cygnus. Its sole planet is one of the smallest exoplanets ever discovered, which is only slightly larger than our own Moon, where it completes a single orbit around its star once every 21.84 hours at a distance of 2.39 million km
**Proposed category:** Exoplanets: K
**Demo Object Name:** KOI-2700
**Author(s):** 星颜Crazy
**Release date:** 2022-Oct-4
**Link to download:** [KOI-2700.zip](https://celestia.space/forum/download/file.php?id=14561&filename=KOI-2700.zip)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/95486841/204442660-42038bf2-54e5-4c47-9a73-3bce9fb35136.jpg)

**Comment by DaveBowman2001** [comment]
**Title:** TrES-5
**Description (Plain Text)**: TrES-5, also known as TOI-3612, is a K-type main sequence star about 1,188 light-years away in the constellation Cygnus. In 2011, a planet larger than Jupiter was found by the Trans-Atlantic Exoplanet Survey (TrES), the fifth and last such world discovered by this survey
**Proposed category:** Exoplanets: T, U and V
**Demo Object Name:** TrES-5
**Author(s):** 星颜Crazy
**Release date:** 2022-Oct-23
**Link to download:** [TrES-5.zip](https://celestia.space/forum/download/file.php?id=14636&filename=TrES-5.zip)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/95486841/204442870-22779e50-9a06-4f09-ba6d-66afc68b1228.jpg)

**Comment by DaveBowman2001** [comment]
**Title:** Kepler-1049
**Description (Plain Text)**: Kepler-1049, also known as KOI-2006, is a red dwarf star about 531.38 light-years away in the constellation Cygnus. A rocky planet about 61% larger than Mars is only 5 million km away from its star, completing an orbit once every 3.27 days
**Proposed category:** Exoplanets: K
**Demo Object Name:** Kepler-1049
**Author(s):** Pan-STARRS
**Release date:** 2022-Oct-30
**Link to download:** [Kepler-1049.zip](https://celestia.space/forum/download/file.php?id=14652&filename=Kepler-1049.zip)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/95486841/204443158-ce968aaa-9bce-4dd9-9db2-af18300370db.jpg)

**Comment by DaveBowman2001** [comment]
Update to the **"65803 Didymos"** addon. Enhanced models, textures and other parameters.

**ID:** [2A604656-CB13-4C35-956F-5F1BAE810396](https://celestia.mobi/resources/item?item=2A604656-CB13-4C35-956F-5F1BAE810396)
**Description:** 65803 Didymos is a 750-m wide near-Earth asteroid belonging to the Apollo group. Discovered in 1996, it has a small moon about 160 m in diameter, named Dimorphos, which is targeted by NASA's Double Asteroid Redirect Test (DART) mission to test planetary defense methods in September 2022.

NOTE: Download the [DART](https://celestia.mobi/resources/item?item=35DD15C3-F872-8561-4302-45B7FD337221) addon to view the simulated impact of the spacecraft onto Dimorphos on 2022-Sept-26

![Untitled2](https://user-images.githubusercontent.com/99847641/203397577-791a94c5-6895-45dd-85cd-72b16f5a5965.jpg)
![Untitled2](https://user-images.githubusercontent.com/99847641/203399495-bb21e962-7ba6-4af7-ac1e-64fc7ae8b95d.jpg)
_The orbit of its moon, Dimorphos, before and after the DART impact, 2022-Sept-26._

**Demo Object Name:** Didymos
**Author(s):** N3cronium, John van Vliet, fyr02, Askaniy
**Last Updated:** 2022-Nov-22
**Link to download:** [65803 Didymos.zip](https://drive.google.com/file/d/199LYqNad_sDyoGCt8siPGxFoU43u_nhT/view?usp=drivesdk)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/99847641/203397577-791a94c5-6895-45dd-85cd-72b16f5a5965.jpg)

**Comment by DaveBowman2001** [comment]
Update to the "**DART**" addon. This is to become fully compatible with the update patch to "65803 Didymos"
 above

**ID:** [35DD15C3-F872-8561-4302-45B7FD337221](https://celestia.mobi/resources/item?item=35DD15C3-F872-8561-4302-45B7FD337221)
**Author(s):** Limax7, John van Vliet, N3cronium
**Last Updated:** 2022-Nov-22
**Link to download:** [DART.zip](https://drive.google.com/file/d/19XtxQ3Ut8rU9SOIZu6bKBNzUPUf4LqRR/view?usp=drivesdk)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/99847641/203401635-7d486d3e-725d-45c9-ac2f-9a394c20059b.jpg)

**Comment by DaveBowman2001** [comment]
Update to the "**DART**" addon. This is to become fully compatible with the update patch to "65803 Didymos" above

**ID:** [35DD15C3-F872-8561-4302-45B7FD337221](https://celestia.mobi/resources/item?item=35DD15C3-F872-8561-4302-45B7FD337221)
**Author(s):** Limax7, John van Vliet, N3cronium
**Last Updated:** 2022-Nov-22
**Link to download:** [DART.zip](https://drive.google.com/file/d/19XtxQ3Ut8rU9SOIZu6bKBNzUPUf4LqRR/view?usp=drivesdk)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/99847641/203401635-7d486d3e-725d-45c9-ac2f-9a394c20059b.jpg)

**Comment by DaveBowman2001** [comment]
Proposal to remove "**Venus Surface Map (Solar System Scope**"

**ID:** [2B0BEAE8-26E1-4CEC-A82D-76650748F9C8](https://celestia.mobi/resources/item?item=2B0BEAE8-26E1-4CEC-A82D-76650748F9C8)
**Author(s):** N/A

**Comment by DaveBowman2001** [comment]
**Title:** Tintin: Explorers on the Moon
**Description (Plain Text):** A scene from "Explorers on the Moon" Tintin is ready to rescue Captain Haddock from the gravitational forces of Adonis.

The scene includes asteroid Adonis, the rocket, Tintin and Captain Haddock.

**Description:** A scene from "Explorers on the Moon" Tintin is ready to rescue Captain Haddock from the gravitational forces of Adonis.

The scene includes asteroid Adonis, the rocket, Tintin and Captain Haddock.
![Untitled2](https://user-images.githubusercontent.com/99847641/202901349-57f4748d-22da-496a-bfd1-c2ba5b75d49e.jpg)
![Untitled2](https://user-images.githubusercontent.com/99847641/202901303-356b335c-2b70-48ae-93b3-d1cfc0d12abb.jpg)
![Untitled2](https://user-images.githubusercontent.com/99847641/202901308-4b4b088d-29af-4203-9647-3b74417fec97.jpg)

**Proposed Category:** Fictional: Crafts & Stations
**Demo Object Name:** Haddock
**Author(s):** lukr
**Release date:** 2005-Sept-1
**Last Updated:** 2013-Jun-4
**Link to download:** [haddock_tintin_rocket.zip](https://drive.google.com/file/d/16yeiju6rmO-7tzsXrLGNlvmFodOseocB/view?usp=drivesdk)
**Image:**
![Untitled2](https://user-images.githubusercontent.com/99847641/202901349-57f4748d-22da-496a-bfd1-c2ba5b75d49e.jpg)

**Comment by DaveBowman2001** [comment]
**Title:** WMAP
**Description (Plain Text)**: The Wilkinson Microwave Anisotropy Probe (WMAP) was a NASA spacecraft that measured the Cosmic Microwave Background radiation from 2001 to 2010. It was placed in a halo orbit around the Sun-Earth Lagrange-2 point, and its findings helped advance our knowledge of the universe itself

**Description (Rich)**: The Wilkinson Microwave Anisotropy Probe (WMAP) was a NASA spacecraft that measured the Cosmic Microwave Background radiation from 2001 to 2010. It was placed in a halo orbit around the Sun-Earth Lagrange-2 point, and its findings helped advance our knowledge of the universe itself

![Untitled](https://user-images.githubusercontent.com/95486841/206858527-77d66b2c-394f-4620-8c3f-a4b07dbc5c77.jpg)

[D-286: WMAP launch (30.6.01) - YouTube](https://www.youtube.com/watch?v=f4zJd1kZhLE)

![Untitled](https://user-images.githubusercontent.com/95486841/206858662-10d476ae-82af-4ecd-b1fd-82b6f833656d.jpg)
_Seven-year WMAP image of cosmic microwave background (CMB) radiation published in July 2010_

**Proposed category:** Spacecraft: Space Telescopes
**Demo Object Name:** WMAP
**Author(s):** DaveBowman2001
**Release date:** 2022-Oct-12
**Link to download:** [WMAP.zip](https://1drv.ms/u/s!Am77jIlPZx-khUKVubaeRqv7COtA?e=o4pQ5Z)
**Image:**
![Untitled](https://user-images.githubusercontent.com/95486841/206858527-77d66b2c-394f-4620-8c3f-a4b07dbc5c77.jpg)

**Comment by SpaceExplorer2008** [comment]
Update (Mercury) https://celestia.mobi/resources/item?item=55D37C62-DC85-852F-9772-5F99EB99CD6D   Updated for Celestia Origin texture.
ID: 55D37C62-DC85-852F-9772-5F99EB99CD6D

**Description (Plain Text):**: This addon adds a new high quality texture to planet Mercury. Taken directly from Celestia Origin.

**Description:** This addon adds a new high quality texture to planet Mercury. Taken directly from Celestia Origin.

![20221111_160246.jpg](https://user-images.githubusercontent.com/99847641/202667629-e5896f50-6b92-4623-8c11-b5ac4e5e48d6.jpg)

![Screenshot_20221111-155126_Celestia.jpg](https://user-images.githubusercontent.com/99847641/202667720-43ff8548-d7de-481d-ae19-9312af2c664a.jpg)

**Author(s):** RVS
**Release date:** 2022-11-11
**Link to download:** https://drive.google.com/file/d/1yDSUNXxP_68vYruKwVJen0FxUgMaTg5c/view?usp=drivesdk
**Image:** ![20221111_160246.jpg](https://user-images.githubusercontent.com/99847641/202667836-01e29dac-c9f7-45c4-86f6-3e20bf6a1c09.jpg)

**Comment by SpaceExplorer2008** [comment]
**Title:** Mercury (Enhanced color)
**Description (Plain Text):** This addon adds, as an alternative surface, a Mercury texture with enhanced color.

**Description:** This addon adds, as an alternative surface, a Mercury texture with enhanced color.

![20221111_162209.jpg](https://user-images.githubusercontent.com/99847641/202668140-4730f4a4-c390-4f68-9086-ddb33aa49aff.jpg)

![20221111_162152.jpg](https://user-images.githubusercontent.com/99847641/202668298-efabef03-814a-4657-beb0-6255cd6c3213.jpg)

**Proposed Category** Solar System: Mercury
**Demo Object Name:**: Mercury
**Author(s):** U.S. Geological Survey
**Release date:** 2022-11-11
**Link to download** https://drive.google.com/file/d/1yDbf8g4A4AwyBGEIOV0N9vIXuBeSfEz5/view?usp=drivesdk
**Image:** ![20221111_162209.jpg](https://user-images.githubusercontent.com/99847641/202668396-fc8c8fd7-96bc-4d23-a922-cc4dfee2e99f.jpg)

**Comment by SpaceExplorer2008** [comment]
**Title:** Venus Surface Map
**Description (Plain Text):** This addon adds a new high quality surface texture for the planet Venus, in false colors.

**Description:** This addon adds a new high quality surface texture for the planet Venus, in false colors.

![20221205_111343.jpg](https://user-images.githubusercontent.com/99847641/205916693-620085b5-895f-449d-84fd-3fbfbf96e307.jpg)
![Screenshot_20221206-093725_Celestia.jpg](https://user-images.githubusercontent.com/99847641/205916722-399ad799-7b79-47c9-85db-6c8c34cd987b.jpg)

**Proposed Category:** Solar System: Venus 
**Demo Object Name:** Venus 
**Author(s):** John van Vliet, CM1215
**Release date:** 2019-08-11
**Link to download:** https://drive.google.com/file/d/1H5X7MHhCrtxVl2g0IYlR3_nXgq6GZRf3/view?usp=drivesdk
**Image:** ![20221205_111343.jpg](https://user-images.githubusercontent.com/99847641/205918417-00b17237-d3cf-46f2-b04e-c6c4afea5bf7.jpg)

**Comment by SpaceExplorer2008** [comment]
**Title:** Venus Topography
**Description (Plain Text):** This addon adds, as an alternative surface, a topographic map for the planet Venus.

**Proposed Category:** Solar System: Venus 
**Demo Object Name:** Venus 
**Author(s):** U.S. Geological Survey
**Release date:** 2022-12-06
**Link to download:** https://drive.google.com/file/d/1H6adsR1nZUUXHFQnAdQHAbRwImktTaii/view?usp=drivesdk
**Image:** ![20221205_111327.jpg](https://user-images.githubusercontent.com/99847641/205922124-d8cc47f8-b0cc-4471-9511-b9d735a6e3f5.jpg)

**Comment by SpaceExplorer2008** [comment]
**Title:** Earth Topography
**Description (Plain Text):** This addon adds, as an alternative surface, a topographic map for the planet Earth.

**Proposed Category:** Solar System: Earth and the Moon
**Demo Object Name:** Earth
**Author(s):** National Oceanic and Atmospheric Administration (NOAA)
**Release date:** 2022-12-07
**Link to download:** https://drive.google.com/file/d/1IEMSx6Py7JKrC5rq99NzHttIGzpz7iNt/view?usp=drivesdk
**Image:** ![20221207_101845.jpg](https://user-images.githubusercontent.com/99847641/206191375-96058d0c-23ae-4590-a293-fece0c39dc5e.jpg)

**Comment by SpaceExplorer2008** [comment]
Proposal to remove [Jupiter's Rings](https://celestia.mobi/resources/item?item=368E4CAC-8C67-4351-454B-1E6BDF416BAF)
ID: 368E4CAC-8C67-4351-454B-1E6BDF416BAF

**Author(s):** N/A

As now Jupiter's rings are in the default version, I believe this addon can be removed.

**Comment by DaveBowman2001** [comment]
**Title:** Luna 15 to 24
**Description (Plain Text)**: The Luna Ye-8-5 spacecraft were a series of Soviet lunar landers that were designed to collect lunar samples robotically and send them back to Earth for analysis. Part of the overall Luna program, they were the first sample-return space probes ever launched and has successfully retrieved a total 301.1 g of lunar material across 3/11 successful missions from 1969 to 1976

**Description (Rich)**: The Luna Ye-8-5 spacecraft were a series of Soviet lunar landers that were designed to collect lunar samples robotically and send them back to Earth for analysis. Part of the overall Luna program, they were the first sample-return space probes ever launched and has successfully retrieved a total 301.1 g of lunar material across 3/11 successful missions from 1969 to 1976
![Untitled](https://user-images.githubusercontent.com/95486841/206859664-65f39e50-ede4-43d2-9fbf-55eded330b66.jpg)
```
<table class="luna-chart">
  <style>.satcom-chart th { text-align:center;position:static  } .luna-chart td { text-align:center }</style>
  <tbody>
    <tr>
      <th colspan="6">Luna Program<br>(1959-Present)</th>
    </tr>
    <tr>
      <th>Mission</th>
      <th>Start</th>
      <th>End</th>
    </tr>
    <tr>
      <td rowspan="2"><strong>Luna 15</strong></td>
      <td>1969-Jul-17</td><td>1969-Jul-21</td>
    </tr>
    <tr>
      <td colspan="2"><i>Crashed on Mare Crisium, notable for running in parallel with Apollo 11</i></td>
    </tr>
    <tr>
      <td rowspan="2"><strong>Cosmos 300</strong></td>
      <td>1969-Sept-23</td><td>1969-Sept-25</td>
    </tr>
    <tr>
      <td colspan="2"><i>Stranded in LEO after TLI burn failure</i></td>
    </tr>
    <tr>
      <td rowspan="2"><strong>Cosmos 305</strong></td>
      <td>1969-Oct-22</td><td colspan="2">1969-Oct-24</td>
    </tr>
    <tr>
      <td colspan="2"><i>Stranded in LEO after TLI burn failure</i></td>
    </tr>
    <tr>
      <td rowspan="2"><strong>Luna 16</strong></td>
      <td>1970-Sept-12</td><td>1970-Sept-24</td>
    </tr>
    <tr>
      <td colspan="2"><i>Landed on Mare Fecundidatis. First successful robotic sample-return mission</i></td>
    </tr>
    <tr>
      <td rowspan="2"><strong>Luna 18</strong></td>
      <td>1971-Sept-2</td><td>1971-Sept-11</td>
    </tr>
    <tr>
      <td colspan="2"><i>Crashed on Terra Apollonius</i></td>
    </tr>
    <tr>
      <td rowspan="2"><strong>Luna 20</strong></td>
      <td>1972-Feb-14</td><td>1972-Feb-25</td>
    </tr>
    <tr>
      <td colspan="2"><i>Landed on Terra Apollonius, just 1.8 km from Luna 18's crash site</i></td>
    </tr>
    <tr>
      <td rowspan="2"><strong>Luna 23</strong></td>
      <td>1974-Oct-28</td><td>1974-Nov-9</td>
    </tr>
    <tr>
      <td colspan="2"><i>Hard-landed on Mare Crisium, failed to complete primary mission</i></td>
    </tr>
    <tr>
      <td rowspan="2"><strong>Luna 24</strong></td>
      <td>1976-Aug-9</td><td>1976-Aug-22</td>
    </tr>
    <tr>
      <td colspan="2"><i>Landed on Mare Crisium, last successful Soviet robotic space mission ever</i></td>
    </tr>
  </tbody>
</table>
```

**Proposed category:** Spacecraft: Moon
**Demo Object Name:** Luna 24
**Author(s):** Jack Higgins, DaveBowman2001
**Release date:** 2008-Jun-7
**Last Updated:** 2022-Dec-1
**Link to download:** [Luna 15 to 24.zip](https://1drv.ms/u/s!Am77jIlPZx-khUPUn_BtjYGpsEHy?e=PlWvlN)
**Image:**
![Untitled](https://user-images.githubusercontent.com/95486841/206859664-65f39e50-ede4-43d2-9fbf-55eded330b66.jpg)

**Comment by SpaceExplorer2008** [comment]
Update (and fixed) [50000 Quaoar](https://celestia.mobi/resources/item?item=79003841-E8FA-6989-927D-98D06F85666B) Added "Replace" parameter for Weywot (Quaoar's moon), as a duplicate of it was being created.

**ID:** 79003841-E8FA-6989-927D-98D06F85666B

**Last Updated**: 2022-11-18
**Link to download:** https://drive.google.com/file/d/14xfd10Ri825eS9_RfULzQeCY98RKyLYE/view?usp=drivesdk

**Comment by SpaceExplorer2008** [comment]
Updated names and description of the following addons:

Update [1950 DA](https://celestia.mobi/resources/item?item=00AD7CD6-F84E-C5CB-A0D5-C96A81CB7D1E)
**ID:** 00AD7CD6-F84E-C5CB-A0D5-C96A81CB7D1E

**Title:** (29075) 1950 DA
**Description (Plain Text):** (29075) 1950 DA is a near-Earth asteroid. It was discovered on February 23, 1950 by Carl A. Wirtanen. A few days after its discovery it was lost, but found again in December 2000. 1950 DA is 1.1 km in diameter. It once had the highest known probability of impacting Earth, but the estimated risk has been updated several times. It is estimated that in the year 2880 the chance of impact on Earth will be 1 in 8,300.

_____________________________

Update [1994 CC](https://celestia.mobi/resources/item?item=56739FD8-D50B-E6FC-AA54-258E4018079D)
**ID:** 56739FD8-D50B-E6FC-AA54-258E4018079D

**Title:** (136617) 1994 CC
**Description (Plain Text):** (136617) 1994 CC is a near-Earth asteroid discovered on February 3, 1994. Radar observations in 2009 allowed the discovery of two moons around the asteroid. It can also be recognized as a trinary asteroid system.

_____________________________

Update [1998 WT24](https://celestia.mobi/resources/item?item=E50FDDEF-E97B-4015-8ED4-F1253B751A31)
**ID:** E50FDDEF-E97B-4015-8ED4-F1253B751A31

**Title:** (33342) 1998 WT24
**Description (Plain Text):** (33342) 1998 WT24 is a near-Earth asteroid and potentially hazardous asteroid of the Aten group. It was discovered on November 25, 1998 by the LINEAR project. The 1998 WT24 orbit makes it cross the orbits of Mercury, Venus and Earth, and due to its low orbit inclination, it is capable of approaching all of these planets. This asteroid measures between 350 and 415 meters in diameter, and if it eventually collided with Earth it would cause devastation on a regional scale.

_____________________________
Update [1999 JV6](https://celestia.mobi/resources/item?item=B942A420-FA0E-0291-D832-3C1E8F79C171)
**ID:** B942A420-FA0E-0291-D832-3C1E8F79C171

**Title:** (85990) 1999 JV6
**Description (Plain Text):** (85990) 1999 JV6 is a near-Earth asteroid. It was discovered on May 13, 1999 by the LINEAR project. Radar images taken by the Arecibo Observatory and Green Bank in 2015 indicate that 1999 JV6 is a contact binary.

_____________________________

Update [2000 DP107](https://celestia.mobi/resources/item?item=1625B1BC-331D-5906-80E2-C2BB56B962C1)
**ID:** 1625B1BC-331D-5906-80E2-C2BB56B962C1

**Title:** (185851) 2000 DP107
**Description (Plain Text):** (185851) 2000 DP107 is a near-Earth asteroid. It was discovered on February 29, 2000 by the LINEAR project. This asteroid also has a moon around it. It can also be recognized as a binary asteroid system. It was the target of the Japanese small probe PROCYON, launched as a secondary payload from Hayabusa2 on December 3, 2014. However, a failure of the PROCYON ion thruster prevented it from reaching the binary asteroid system.

**Comment by SpaceExplorer2008** [comment]
Update [Earth's History](https://celestia.mobi/resources/item?item=F86AC158-FD5E-409E-83A1-6FEB3120F4F8)
Small improvement in the description.

**ID:** F86AC158-FD5E-409E-83A1-6FEB3120F4F8

**Description (Plain Text):** An addon that brings a spectacular view of the history of our planet Earth, from 4.5 billion to 20 million years in the past. This addon adds 46 alternate surfaces for Earth, showing what our planet might have looked like at different times.

**Description:** An addon that brings a spectacular view of the history of our planet Earth, from 4.5 billion to 20 million years in the past. This addon adds 46 alternate surfaces for Earth, showing what our planet might have looked like at different times.

NOTES:

- If you install this addon, it is highly recommended that you also install the  [Earth](https://celestia.mobi/resources/item?item=5338AEF4-EA7E-49C8-A4D0-FF1297F8182F) addon, so that the default Earth texture matches these alternate surfaces.

- When viewing the alternate surface "4500 million years ago", you must disable the atmosphere and clouds in Celestia's settings.

**Comment by DaveBowman2001** [comment]
> Updated names and description of the following addons:

For these updates, lev requested to have their IDs listed on the bottom so that he can update them easily

**Comment by SpaceExplorer2008** [comment]
**Title:** (8567) 1996 HW1
**Description (Plain Text):** (8567) 1996 HW1 is a near-Earth asteroid discovered on April 23, 1996. In 2008, observations made with the Arecibo
Observatory revealed that it is a very elongated object and a contact binary.

**Proposed Category:** Asteroids & NEOs: Near-Earth Objects
**Demo Object Name:** 1996 HW1
**Author(s):** Greg Frieger, Art Blos
**Release date:** 2022-11-18
**Link to download:** https://drive.google.com/file/d/14mIP2GWcic0TxwUwUeYANSKF6ooEKmBf/view?usp=drivesdk
**Image:** ![20221118_180441.jpg](https://user-images.githubusercontent.com/99847641/202803148-faecc97e-8d41-4db8-9aeb-77bcaccf46a5.jpg)

**Comment by SpaceExplorer2008** [comment]
Update [Moshup-Squannit](https://celestia.mobi/resources/item?item=3BD0DE85-37C2-E52F-EE6E-CC220DE711B9)
ID: 3BD0DE85-37C2-E52F-EE6E-CC220DE711B9

Title: 66391 Moshup
Description (Plain Text): 66391 Moshup (1999 KW4) is a near-Earth binary asteroid of the Aten group. It was discovered on May 20, 1999 by the LINEAR project. The smaller asteroid in the system, Squannit, is also often considered a small moon of the larger asteroid.

**Comment by SpaceExplorer2008** [comment]
> > Updated names and description of the following addons:
> 
> For these updates, lev requested to have their IDs listed on the bottom so that he can update them easily
> 
> For example:
> ```
> 2B0BEAE8-26E1-4CEC-A82D-76650748F9C8
> 00A49425-F9E4-9E59-B578-27DDC0B3FE3D
> F86AC158-FD5E-409E-83A1-6FEB3120F4F8
> 1C96B314-2B26-D39A-7122-7AB98EFDB3B0
> ```

How do I do that?

**Comment by SpaceExplorer2008** [comment]
**Title:** (52760) 1998 ML14
**Description (Plain Text):** (52760) 1998 ML14 is a near-Earth asteroid about 1 km in diameter. It was discovered on June 24, 1998 by the LINEAR project. Soon after its discovery, it was observed by the Arecibo and Goldstone observatories.
**Proposed Category:** Asteroids & NEOs: Near-Earth Objects
**Demo Object Name:** 1998 ML14
**Author(s):** Greg Frieger, Art Blos
**Release date:** 2022-12-02
**Link to download:** https://drive.google.com/file/d/1EyBHD2doynVGUMoLPfjkbLpm0VT1pROy/view?usp=drivesdk
**Image:** ![20221202_114059.jpg](https://user-images.githubusercontent.com/99847641/205319296-78c5964a-5362-4778-a163-a2bea61e15d0.jpg)

**Comment by SpaceExplorer2008** [comment]
**Title:** (162421) 2000 ET70
**Description (Plain Text):** (162421) 2000 ET70 is a near-Earth asteroid discovered on March 8, 2000, by the LINEAR project. It is about 2.2 km in diameter. During a close approach to Earth in February 2012, it was observed by the Arecibo and Goldstone observatories, which made it possible to determine the asteroid's rotation and shape.

**Proposed Category:** Asteroids & NEOs: Near-Earth Objects
**Demo Object Name:** 2000 ET70
**Author(s):** Greg Frieger, Art Blos 
**Release date:** 2022-12-05
**Link to download:** https://drive.google.com/file/d/1Exo3tdFOAh9Ee7vYctQIdAzGVqBc766k/view?usp=drivesdk
**Image:** ![20221202_114041.jpg](https://user-images.githubusercontent.com/99847641/205648115-f77859d9-4d9c-452d-af46-0cb66443085b.jpg)

**Comment by SpaceExplorer2008** [comment]
**Title:** 2018 LA
**Description (Plain Text):** 2018 LA was a small Apollo asteroid about 2.6-3.8 m (9-12 ft) that entered Earth's atmosphere on June 2, 2018, near the border of Botswana and South Africa. It was discovered just 8 hours before impact by the Mount Lemmon Survey.

NOTE: To see asteroid 2018 LA you must set the time in Celestia before 2018-Jun-02, 16:56:30 UTC

**Proposed Category:** Asteroids and NEOs: Near-Earth Objects
**Demo Object Name:** 2018 LA
**Author(s):** Art Blos
**Release date:** 2022-12-06
**Link to download:** https://drive.google.com/file/d/1EvCbmbT5Uxhx-mpNfFcYRW_y7K6M__KY/view?usp=drivesdk
**Image:** ![20221202_114126.jpg](https://user-images.githubusercontent.com/99847641/205923486-5ff8b020-255d-4cec-b3a8-f3429ada9951.jpg)

**Comment by SpaceExplorer2008** [comment]
`28228`

**Comment by SpaceExplorer2008** [comment]
> > Updated names and description of the following addons:
> 
> For these updates, lev requested to have their IDs listed on the bottom so that he can update them easily
> 
> For example:
> ```
> 2B0BEAE8-26E1-4CEC-A82D-76650748F9C8
> 00A49425-F9E4-9E59-B578-27DDC0B3FE3D
> F86AC158-FD5E-409E-83A1-6FEB3120F4F8
> 1C96B314-2B26-D39A-7122-7AB98EFDB3B0
> ```

Now I learned how to do this. But for individual updates, it's not necessary, right? Only for multiple updates

**Comment by DaveBowman2001** [comment]
Yes, only for multiple updates

**Comment by SevenSpheres** [comment]
Updated description of [2MASS J1119-1137](https://celestia.mobi/resources/item?item=D063E245-E5A4-AAC9-FDD4-39328ECD5D3E) for clarity, and removed statement that the Unconfirmed Exoplanets addon is a dependency since this addon fully replaces the candidate satellite.

**ID:** D063E245-E5A4-AAC9-FDD4-39328ECD5D3E
**Description (Plain Text)**: 2MASS J11193254-1137466, also known as TWA 42, is a planetary-mass binary consisting of two young L-type rogue planets or sub-brown dwarfs, about 86.1 light-years away in the constellation Crater. A single super-Earth-sized satellite may be orbiting the primary within its current "habitable zone", but its existence has yet to be fully confirmed.

**Comment by DaveBowman2001** [comment]
**Title**: Project Apollo

**Comment by DaveBowman2001** [comment]
**Title**: Tranquility Base

**Comment by DaveBowman2001** [comment]
**Title**: Statio Cognitum

**Comment by DaveBowman2001** [comment]
**Title**: Fra Mauro Highlands

**Comment by DaveBowman2001** [comment]
**Title**: Hadley-Apennine

**Comment by DaveBowman2001** [comment]
**Title**: Descartes Highlands

**Comment by DaveBowman2001** [comment]
**Title**: Polar Operational Environmental Satellite
**Proposed category**: Spacecraft: Earth Science
**Demo Object**: NOAA-19
**Author(s)**: DaveBowman2001, toutatis, Gironde
**Last updated**: 2022-Oct-29

**Comment by DaveBowman2001** [comment]
**Title**: Chongfu Shiyong Shiyan Hangtian Qi
**Description (Plain Text)**: The Chongfu Shiyong Shiyan Hangtian Qi (CSSHQ) is the first Chinese reusable spacecraft. Little is known about the spacecraft itself due to its military nature, but unofficial reports indicate that it is a spaceplane of the Shenlong program which it is claimed that it shares similar designs and purpose with the Boeing X-37B

**Description**: The Chongfu Shiyong Shiyan Hangtian Qi (CSSHQ) is the first Chinese reusable spacecraft. Little is known about the spacecraft itself due to its military nature, but unofficial reports indicate that it is a spaceplane of the Shenlong program which it is claimed that it shares similar designs and purpose with the [Boeing X-37B](https://celestia.mobi/resources/item?item=E2BBC744-C70A-85D0-C3EF-DFD7D2AA4E35)

**Proposed category**: Spacecraft: Technology Demonstrators
**Demo Object**: CSSHQ-2
**Author(s)**: DaveBowman2001
**Release date**: 2022-Nov-19

**Comment by DaveBowman2001** [comment]
**Title**: Intelsat-VI
**ID Requirement**: 8
**Description**: The Intelsat VI-series were the sixth generation communications satellites built by the Hughes Aircraft Company for the Intelsat Corporation. From 1989 to 1991, a total of five satellites, each weighing 4,330 kg, were launched into GEO. One satellite (Intelsat 603) was left stranded in LEO after a launch failure in 1990, where it was subsequently rescued during the maiden flight of Space Shuttle Endeavour in May 1992
**Proposed category**: Spacecraft: Communications Satellites
**Demo Object**: Intelsat 603
**Author(s)**: DaveBowman2001
**Release date**: 2022-Nov-24

**Comment by DaveBowman2001** [comment]
Update to the **Tiangong Space Station** addon. Added the temporary versions of the two expansion modules, Mengtian and Wentian, ~~so that it won't get barraged by impatient commenters anymore~~ while we wait for Tom's proper updates for this addon lol

**ID**: [BA5FBDDC-FEEA-F437-FC8E-1DF31AD6E4FB](https://celestia.mobi/resources/item?item=BA5FBDDC-FEEA-F437-FC8E-1DF31AD6E4FB)
**Link to download**: 
**Image**:

**Comment by DaveBowman2001** [comment]
**Title**: Project Apollo
**ID Requirement**: A
**Proposed category**: Spacecraft: Crewed/Resupply Missions
**Demo Object**: Eagle LEM
**Author(s)**: DaveBowman2001, ElChristou, Endeavor35, John van Vliet, Max Grüeter, linuxman
**Last updated**: 2022-Dec-11
**Link to download**:

**Comment by DaveBowman2001** [comment]
**Title**: Apollo-Soyuz Test Project
**ID Requirement**: 45
**Description (Plain Text)**: The Apollo-Soyuz Test Project (ASTP) is the first crewed international space mission in history, carried out jointly by the United States and the Soviet Union in July 1975. It symbolized the end of the Space Race that started with the launch of Sputnik 1 in 1957 and reached its peak during Apollo 11 in 1969. ASTP marked the last flight of the Apollo Program, with the US portion sometimes being referred to (unofficially) as Apollo 18. The mission's legacy on peaceful cooperation in space is still felt today aboard the International Space Station since 1998

**Description**: The Apollo-Soyuz Test Project (ASTP) is the first crewed international space mission in history, carried out jointly by the United States and the Soviet Union in July 1975. It symbolized the end of the Space Race that started with the launch of Sputnik 1 in 1957 and reached its peak during Apollo 11 in 1969. ASTP marked the last flight of the Apollo Program, with the US portion sometimes being referred to (unofficially) as Apollo 18. The mission's legacy on peaceful cooperation in space is still felt today aboard the International Space Station since 1998

NOTES:
- Requires the Project Apollo addon installed in order to see the full ASTP mission in action. Without it, by default only the Soyuz missions are visible on this addon
- Also includes the three precursor test flights and one post-ASTP mission of the Soyuz 7K-TM spacecraft (see Table below)

![Untitled](https://user-images.githubusercontent.com/95486841/207517727-36670c82-74c6-4c44-9a49-eaf99ed81044.jpg)
_Apollo-Soyuz, also known as Soyuz-Apollo in the USSR, docked together on July 1975_

![Untitled](https://user-images.githubusercontent.com/95486841/207517842-090f380e-c4f4-4df8-961e-495f7d432fee.jpg)
_Apollo 18_

![Untitled](https://user-images.githubusercontent.com/95486841/207518176-4a98a1ed-5218-418f-95ba-9a1609bf044b.jpg)
_Soyuz 19_

```
<table class="astp-chart">
  <style>.satcom-chart th { text-align:center;position:static  } .astp-chart td { text-align:center }</style>
  <tbody>
    <tr>
      <th colspan="6">Apollo-Soyuz<br>(1974-1975)</th>
    </tr>
    <tr>
      <th colspan="2">Mission</th>
      <th>Start</th>
      <th>End</th>
    </tr>
    <tr>
      <td colspan="2" rowspan="2"><strong>Cosmos 638</strong></td>
      <td>1974-Apr-3</td><td>1974-Apr-13</td>
    </tr>
    <tr>
      <td colspan="2"><i>Unmanned test flight of the Soyuz 7K-TM</i></td>
    </tr>
    <tr>
      <td colspan="2" rowspan="2"><strong>Cosmos 672</strong></td>
      <td>1974-Aug-12</td><td>1974-Aug-18</td>
    </tr>
    <tr>
      <td colspan="2"><i>Unmanned test flight of the Soyuz 7K-TM</i></td>
    </tr>
    <tr>
      <td colspan="2" rowspan="2"><strong>Soyuz 16</strong></td>
      <td>1974-Dec-2</td><td colspan="2">1974-Dec-8</td>
    </tr>
    <tr>
      <td colspan="2"><i>Crewed test flight/dress rehearsal before ASTP</i></td>
    </tr>
    <tr>
      <td rowspan="3"><strong>Apollo-Soyuz Test Project (ASTP)</strong></td><td><strong>Soyuz 19</strong></td>
      <td rowspan="2">1975-Jul-15</td><td>1975-Jul-21</td>
    </tr>
    <tr>
      <td><strong>Apollo 18</strong><td>1975-Jul-24</td>
    </tr>
    <tr>
      <td colspan="3"><i>First international crewed spaceflight in history</i></td>
    </tr>
    <tr>
      <td colspan="2" rowspan="2"><strong>Soyuz 22</strong></td>
      <td>1976-Sept-15</td><td>1976-Sept-23</td>
    </tr>
    <tr>
      <td colspan="2"><i>Final non-space station crewed mission of the Soyuz spacecraft</i></td>
    </tr>
  </tbody>
</table>
```
_Timeline of the ASTP program and related spaceflights, 1974-1975_

![Untitled](https://user-images.githubusercontent.com/95486841/207520888-d73e4f65-e0d5-440d-95db-6842ca8ce93f.jpg)
_The two crews of Apollo-Soyuz: (Top) Tom Stafford and Alexei Leonov, (Below) Deke Slayton, Vance Brand and Valery Kubasov_

**Proposed category**: Spacecraft: Crewed/Resupply Missions
**Demo Object**: ASTP
**Author(s)**: DaveBowman2001, linuxman
**Release date**: 2006-Sept-14
**Last updated**: 2022-Dec-11
**Link to download**: [Apollo-Soyuz Test Project.zip](https://github.com/levinli303/celestia.mobi/files/10224763/Apollo-Soyuz.Test.Project.zip)
**Image**:
![Untitled](https://user-images.githubusercontent.com/95486841/207517727-36670c82-74c6-4c44-9a49-eaf99ed81044.jpg)

**Comment by DaveBowman2001** [comment]
**Title**: INSAT-3E
**Description (Plain Text)**: INSAT-3E is the fourth satellite to be launched in the INSAT-3 series. It is an Indian communications satellite designed for providing high-speed communication, television, VSAT & tele-education services. It was launched in 2003 and it continued operations in GEO until it was decommissioned in April 2014
**Proposed category**: Spacecraft: Communications Satellites
**Demo Object**: INSAT-3E
**Author(s)**: toutatis
**Release date**: 2018-Jul-18
**Link to download**: [INSAT-3E.zip](https://github.com/levinli303/celestia.mobi/files/10226346/INSAT-3E.zip)
**Image**:
![Untitled](https://user-images.githubusercontent.com/95486841/207559831-2dae5fff-7e8d-413c-8b60-7bf03a16e83b.jpg)

**Comment by DaveBowman2001** [comment]
**Title**: INSAT-3E
**Description (Plain Text)**: INSAT-3E is the fourth satellite to be launched in the INSAT-3 series. It is an Indian communications satellite designed for providing high-speed communication, television, VSAT & tele-education services. It was launched in 2003 and it continued operations in GEO until it was decommissioned in April 2014
**Proposed category**: Spacecraft: Communications Satellites
**Demo Object**: INSAT-3E
**Author(s)**: toutatis
**Release date**: 2018-Jul-18
**Link to download**: [INSAT-3E.zip](https://github.com/levinli303/celestia.mobi/files/10226346/INSAT-3E.zip)
**Image**:
![Untitled](https://user-images.githubusercontent.com/95486841/207559831-2dae5fff-7e8d-413c-8b60-7bf03a16e83b.jpg)

**Comment by SpaceExplorer2008** [comment]
@DaveBowman2001 Why don't you put Apollo 18 in the Apollo-Soyuz Test Project addon itself?

**Comment by DaveBowman2001** [comment]
It has a hard-coded situation similar to the STS-2022 satellites. Also, the ASTP addon's contents are focused on the Russian side of the mission anyway...simply putting Apollo 18 there would only balloon its size since models cannot be defined by an SSC file unless it belongs to the same parent folder

**Comment by SpaceExplorer2008** [comment]
**Title:** POTS-1:
**Description (Plain Text):** POTS-1b is a gas giant orbiting the star POTS-1, which is 3913.9 light-years away from Earth. It has a mass of 2.31 Jupiters and orbits its star every 3.2 days. The discovery of this exoplanet was announced in 2013.
**Proposed Category:** Exoplanets: Confirmed: (P)
**Author(s):** 2022-11-12
**Link to download:** https://drive.google.com/file/d/1M_I4axa6y6dbMj__l0K53jO5dLACvN8E/view?usp=drivesdk
**Image:** ![20221214_095340.jpg](https://user-images.githubusercontent.com/99847641/207600470-889e5235-3063-4faa-a025-2abeb4c2b510.jpg)

**Comment by SpaceExplorer2008** [comment]
**Title:** TAP-26
**Description**: TAP-26b is a gas giant orbiting the star TAP-26, located 396 light-years away from Earth. It has a mass of 1.66 Jupiters and its orbital period is 10.8 days. The discovery of this exoplanet was announced in 2017.
**Proposed Category:** Exoplanets: Confirmed (T, U and V)
**Author(s): Gurren Lagann**
**Release date:** 2022-11-26
**Link to download:** https://drive.google.com/file/d/1MYyhG28H-VJlZEEeH7VmGOrcOQzsuvl0/view?usp=drivesdk
**Image:** ![20221214_095624.jpg](https://user-images.githubusercontent.com/99847641/207601798-5b9f63c0-2b92-4bb0-8dc1-25d1cce17b68.jpg)

**Comment by levinli303** [comment]
please avoid making new submissions when there are too many pending. I have limited time and it is not really good to have this list always growing since there might be critical changes that are more worthy of being up there before new add-ons.

So instead of making new ones as soon as they are available. make sure they SHOULD be on the website and prioritize when there are too many.

**Comment by levinli303** [comment]
And for Rich description, you are encouraged to provide HTML directly

**Comment by levinli303** [comment]
last uploaded: Kepler-1049

**Comment by levinli303** [comment]
all uploaded

**Comment by DaveBowman2001** [comment]
Closing for now....
